common/changes/@uifabric/utilities/lingge-persona-customize-image_2017-09-21-03-12.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
```

Comment:
future reference:

`rush change -b upstream/master`

This will eval your changes against "master" branch of your "upstream" remote.

Reply:
Thank you, will do it in my update.